api_tester/lib/api_tests/purchases_flutter_api_test.dart

```diff
@@ -72,42 +72,49 @@ class _PurchasesFlutterApiTest {
   void _checkPurchaseProduct() async {
     String productIdentifier = "fakeProductId";
     UpgradeInfo? upgradeInfo;
+    GoogleProductChangeInfo? googleProductChangeInfo;
     PurchaseType purchaseType = PurchaseType.subs;
     CustomerInfo customerInfo = await Purchases.purchaseProduct(
         productIdentifier,
         upgradeInfo: upgradeInfo,
-        isPersonalizedPrice: true);
+        googleIsPersonalizedPrice: true);
     customerInfo = await Purchases.purchaseProduct(productIdentifier,
         upgradeInfo: upgradeInfo, type: purchaseType);
+    customerInfo = await Purchases.purchaseProduct(productIdentifier,
+        googleProductChangeInfo: googleProductChangeInfo);
     customerInfo = await Purchases.purchaseProduct(productIdentifier,
         upgradeInfo: upgradeInfo);
     customerInfo = await Purchases.purchaseProduct(productIdentifier,
-        isPersonalizedPrice: true);
+        googleIsPersonalizedPrice: true);
     customerInfo = await Purchases.purchaseProduct(productIdentifier);
   }
 
   void _checkPurchasePackage(Package package) async {
     UpgradeInfo? upgradeInfo;
+    GoogleProductChangeInfo? googleProductChangeInfo;
     CustomerInfo customerInfo =
         await Purchases.purchasePackage(package, upgradeInfo: upgradeInfo);
     customerInfo = await Purchases.purchasePackage(package,
-        upgradeInfo: upgradeInfo, isPersonalizedPrice: true);
-    customerInfo =
-        await Purchases.purchasePackage(package, isPersonalizedPrice: true);
+        googleProductChangeInfo: googleProductChangeInfo,
+        googleIsPersonalizedPrice: true);
+    customerInfo = await Purchases.purchasePackage(package,
+        upgradeInfo: upgradeInfo, googleIsPersonalizedPrice: true);
+    customerInfo = await Purchases.purchasePackage(package,
+        googleIsPersonalizedPrice: true);
   }
 
-  void _checkPurchaseSubscriptionOption(
-      SubscriptionOption subscriptionOption, UpgradeInfo? upgradeInfo) async {
+  void _checkPurchaseSubscriptionOption(SubscriptionOption subscriptionOption,
```

Comment:
this rename sooooort of makes me wonder if we should reanme to `GoogleSubscriptionOption` too 🙈

Reply:
I really want to 😬 Probably do that during the great module migration whenever that happens 😇